cluster-provision/k8s/1.20/manifests/cni.diff

```diff
@@ -14,6 +14,18 @@
            },
            "policy": {
                "type": "k8s"
+@@ -3642,6 +3642,11 @@ spec:
+             # Auto-detect the BGP IP address.
+             - name: IP
+               value: "autodetect"
++            # Use IP/IPv6 addresses on ifaces eth0, eth1, eth2, etc.
++            - name: IP_AUTODETECTION_METHOD
++              value: "interface=eth.*"
++            - name: IP6_AUTODETECTION_METHOD
++              value: "interface=eth.*"
+             # Enable IPIP
+             - name: CALICO_IPV4POOL_IPIP
+               value: "Always"
```

Comment:
And this one? Why are you updating it ?

Reply:
I thought that I messed up, but it's just so confusing to see diff in a diff.

The lines I'm adding here are:
```
@@ -3642,6 +3642,11 @@ spec:
             # Auto-detect the BGP IP address.
             - name: IP
               value: "autodetect"
+            # Use IP/IPv6 addresses on ifaces eth0, eth1, eth2, etc.
+            - name: IP_AUTODETECTION_METHOD
+              value: "interface=eth.*"
+            - name: IP6_AUTODETECTION_METHOD
+              value: "interface=eth.*"
             # Enable IPIP
             - name: CALICO_IPV4POOL_IPIP
               value: "Always"
```

So in Github diff, only the lines with two ++ are being added, the lines with one + are just part of the diff context

Reply:
Inception 0_o .

Thanks for explaining.